### ISSUE TITLE
"Search here" button on map is not keyboard operable

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#669]
On the home page and search results, there's something odd with the map search.

<img width="240" alt="Screen Shot 2020-06-23 at 3 37 22 PM" src="https://user-images.githubusercontent.com/6945691/85473225-c59ed080-b567-11ea-88be-a9e00f0306ca.png">

<img width="240" alt="Screen Shot 2020-06-23 at 3 39 26 PM" src="https://user-images.githubusercontent.com/6945691/85473217-c33c7680-b567-11ea-8903-8524f13d4656.png">

1. The `<div>` that contains the map has `tabindex="0"`, which puts the `<div>` in the tab order - but it is reported as an error, because the `<div>` is not an operable element. This should be removed.

2. The Search button itself is not focusable. After the `<div>`, taborder then goes to Zoom in, then Zoom out, then skips to the "leaflet" links at the bottom of the map. If I force focus to the button, it's not operable. (In the HTML it's an `<a>` with no `href` so it's not operable by default.) Note that the checkbox ("Search when I move the map") is automatically focusable/operable. Replacing the `<a>` with `<button>` also makes it focusable/operable, but it doesn't actually work when it's activated.

3. The `<a>` also has an inline style of ` outline: none;` which removes its visible focus indicator. This needs to be removed.

**Comment by cbeer** [comment]
> The <div> that contains the map has tabindex="0", which puts the <div> in the tab order - but it is reported as an error, because the <div> is not an operable element. This should be removed.


It looks like when you have focus on the `div`, you can use arrow keys to pan the map around.

**Comment by mejackreed** [comment]
Yes and +/- keys will also zoom.

**Comment by jvine** [comment]
> It looks like when you have focus on the div, you can use arrow keys to pan the map around.

Ah interesting. This is not announced in any way, it's just a mysterious tab stop. I'll look into it further.

**Comment by jvine** [comment]
Note that items 2 and 3 above are not waiting on any design/investigation and should be actionable? Assuming we can get at it. This feels like a priority fix because it's disabling functionality that isn't available to a keyboard user in any other way.

**Comment by jvine** [comment]
Looking further into item 1:

If we can modify the `<div>` as follows:
- add `aria-label="Zoomable map"
- remove `outline: none`

and remove `.leaflet-container {outline: 0;}`

then it has a useful effect for keyboard and screen readers. Keyboard user can see where they are; SR user no longer gets a mystery tab stop; SR menu user can jump to the map directly.
